depmap/0.download_depmap.ipynb

```diff
@@ -0,0 +1,389 @@
+    "\n",
+    "print(depmap_df.shape)\n",
+    "depmap_df.head(3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "b38e1745-9e73-4bee-bb5d-163457b45c75",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(17386, 3)\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>depmap_column_name</th>\n",
+       "      <th>gene_symbol</th>\n",
+       "      <th>ncbi_entrez_id</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>A1BG (1)</td>\n",
+       "      <td>A1BG</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>A1CF (29974)</td>\n",
+       "      <td>A1CF</td>\n",
+       "      <td>29974</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>A2M (2)</td>\n",
+       "      <td>A2M</td>\n",
+       "      <td>2</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  depmap_column_name gene_symbol ncbi_entrez_id\n",
+       "0           A1BG (1)        A1BG              1\n",
+       "1       A1CF (29974)        A1CF          29974\n",
+       "2            A2M (2)         A2M              2"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# The columns are of the format: symbol (NCBI Entrez ID)\n",
+    "# Transform them and write out the dictionary\n",
+    "genes = depmap_df.columns[1:].tolist()\n",
+    "\n",
+    "gene_data = []\n",
+    "for gene in genes:\n",
+    "    gene_name, ncbi_entrez_gene = gene.split(\" \")\n",
+    "    gene_data.append([gene, gene_name, ncbi_entrez_gene.strip(\"()\")])\n",
+    "\n",
+    "gene_df = pd.DataFrame(\n",
+    "    gene_data, columns=[\"depmap_column_name\", \"gene_symbol\", \"ncbi_entrez_id\"]\n",
+    ")\n",
+    "\n",
+    "gene_df.to_csv(output_gene_dictionary, sep=\"\\t\", index=False)\n",
+    "\n",
+    "print(gene_df.shape)\n",
+    "gene_df.head(3)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
It looks like this might have been run from outside the `drug-viability` environment.

Reply:
thanks for pointing this out. Not sure how this keeps happening in this repo! will fix